lib/read.js

```diff
@@ -217,6 +238,15 @@ class ReadSql extends AdapterRead {
         } else {
             query = query.offset(this.offsetCount);
         }
+        
+        if (this.sortBorderValue) {
+            query = query.where(
+                this.sortFields[0].field,
+                this.sortFields[0].direction === 'desc' ? '<=' : '>=',
```

Comment:
How are we making sure we don't get duplicate points if there's a bunch of points with `sortBorderValue` as their value spanning two pages?